raiden_contracts/tests/test_deploy_data.py

```diff
@@ -48,31 +50,65 @@ def reasonable_deployment_of_a_contract(deployed):
     assert isinstance(deployed['constructor_arguments'], list)
 
 
+RAIDEN_CONTRACT_NAMES = {'EndpointRegistry', 'TokenNetworkRegistry', 'SecretRegistry'}
```

Comment:
Why not make this into a tuple?

Reply:
Because I don't care about the order.  I can just tuple it, too (maybe the test will be more deterministic).

Reply:
Tupled.